lldb/packages/Python/lldbsuite/test/gdbclientutils.py

```diff
@@ -86,8 +88,8 @@ class MockGDBServerResponder:
     handles any packet not recognized in the common packet handling code.
     """
 
-    registerCount = 40
-    packetLog = None
```

Comment:
This predates you, but I wonder why we even have a class level attribute for this? Maybe I am misunderstanding some aspect of Python because it seems to me that the packet log should never be None. Empty list, sure, but not None.

Reply:
Assuming it passes testing, https://github.com/llvm/llvm-project/pull/162453 will remove this line.